burn-core/src/nn/lstm/lstm.rs

```diff
@@ -219,16 +217,16 @@ impl<B: Backend> Lstm<B> {
 
 #[cfg(test)]
 mod tests {
-
     use super::*;
     use crate::{module::Param, nn::LinearRecord, TestBackend};
     use burn_tensor::Data;
 
     #[test]
-    fn initializer_default() {
```

Comment:
We should probably note with a TODO to test with default initializer (Xavier). @agelas is currently working in this area, perhaps he could add a test. At least we know we don't have the tests for the default initializer.